Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -718,6 +718,8 @@ def identitySim(wf):
 steps['TTbar_13_94Xv1NanoAODINPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13/CMSSW_9_4_0_pre3-PU25ns_94X_mc2017_realistic_v4-v1/MINIAODSIM',label='nanoaod94X',location='STD')}
 steps['TTbar_13_94Xv2NanoAODINPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13/CMSSW_9_4_5_cand1-94X_mc2017_realistic_v14_PU_RelVal_rmaod-v1/MINIAODSIM',label='nanoaod94Xv2',location='STD')}
 
+steps['TTbar_13_106Xv1NanoAODINPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13/CMSSW_10_6_14-106X_mc2017_realistic_v7-v1/MINIAODSIM',label='nanoaod106X',location='STD')}
```

Comment:
@mariadalfonso Is it better to use PU dataset? or doesn't matter?
/RelValTTbar_13/CMSSW_10_6_14-PU25ns_106X_mc2017_realistic_v7-v1/MINIAODSIM

Reply:
For now I would say it doesn't matter, since those are intended mainly for technical tests.
Later we might want to rationalise all input for the EOY, UL, reminiUL  and for data and PU.

Reply:
Right. Everything in runTheMatrix is for technical test and I can tell that no one will come back to this again so i would propose to update to PU dataset as it existed and to avoid any confusion in the future since other similars workflows were using PU datasets. Thanks.

Reply:
PU samples updated